client/state/reader/posts/normalization-rules.js

```diff
@@ -78,7 +78,7 @@ const hasShortContent = ( post ) => getCharacterCount( post ) <= PHOTO_ONLY_MAX_
  * @returns {Object}            The classified post
  */
 export function classifyPost( post ) {
-	const imagesForGallery = filter( post.content_images, imageIsBigEnoughForGallery );
+	const imagesForGallery = getImagesFromPostToDisplay( post, GALLERY_MIN_IMAGES + 1 );
```

Comment:
Other than the bug being fixed, everything in my testing works the same... but im a bit confused by the min images const and the +1.

* Why the +1 ?
* Why is this a min images constant and not a max images constant? it seems to be used as the cutoff index to limit what is returned from that function, so it reads weird that the cutoff is near the minimum as opposed to the max?

As mentioned there don't seem to be any regressions, so i could be easily misunderstanding something here.

Approving given the issue is fixed and im not seeing any regressions.

Reply:
> Why the +1 ?

True, I think I just got this working that way and left it the way it was, it makes no sense conceptually! 😆 In the other code, we just use the number `10`, I've added a new constant `GALLERY_MAX_IMAGES = 10` and used that in both places now.

> Why is this a min images constant and not a max images constant? good point, that would make more sense 👍